### PR TITLE
Finished main section and header

### PR DESCRIPTION
**Pull request by iagopiresdev**
- Refactored pages and root to remove unnecessary css completely.
- Updated header and added sandwich for mobile.
- Added custom fonts.
- Finished main page according to the Figma design.
- Added framer-motion dependency.
- Added animations for “setas” and the main circle.

### CODE REVIEW

**Comment by guilhermemdasilva** [comment]
- responsive design, works well on most mobiles;
- was the footer removed as intended or by mistake?
- on very small screen, white letters mix with white background, but shouldn't be an issue for now;
- IMHO: overlapping FF1C00 and 000000 hurts the eyes a bit, I'd rather take someone else's opinion;
- Menu items seem a bit dull. Maybe use Redonda instead of Raleway

File: src/App.jsx [hunk 1]
```diff
@@ -8,10 +8,10 @@ function App() {
 
   return (
     <Router>
-      <header className='bg-[#F4F9F2]'>
```
Comment (iagopiresdev) [inline src/App.jsx]: Changed bg

File: src/Pages/QuemSomos.jsx [hunk 2]
```diff
@@ -4,42 +4,14 @@ import arrowRight from "../assets/main_arrow_right.svg";
 import arrowLeft from "../assets/main_arrow_left.svg";
 import circle from "../assets/main_circle.svg";
 import hat from "../assets/main_hat.svg";
+import ComoParticipar from "./ComoParticipar";
 
 function QuemSomos() {
   return (
-    <section className="flex flex-col items-center justify-center sm:mt-[-40px] mt-[-30px] z-0 overflow-hidden w-full">
```
Comment (iagopiresdev) [inline src/Pages/QuemSomos.jsx]: Deleted arrows

**Review by iagopiresdev — COMMENTED** [review]
- Changed bg color to make it easier on the eyes (#DEF2D6).
- Footer was removed for a multi-page design, but I personally think that a single page application would suit better the project, so probably it will be brought back later.
- Added support for 2XL devices, so the cicle doesn't become super tiny.
- The menu items font isn't very pleasing with Raleway in my opinion.
- Removed arrows for excessive amout of bugs, I'll need them done as PNG so they don't break the page's design/structure.

**Comment by guilhermemdasilva** [comment]
looks good to me

**Review by guilhermemdasilva — APPROVED** [review]
looks good to me

**Pull request merged**